server/models/Request.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const { Schema, model } = mongoose;
+
+const requestSchema = new mongoose.Schema(
+  {
+    owner: { type: Schema.Types.ObjectId, ref: "User", required: true },
```

Comment:
Optional: Do you mind reformatting this to use multi-line instead of all on one line? It will allow refactoring to be easier in the future. 
```
  owner: {
    type: Schema.Types.ObjectId,
    ref: "User",
    required: true,
  },
```

Reply:
Yes, no problem. Prettier accepted both ways 😅